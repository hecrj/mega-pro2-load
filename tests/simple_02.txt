14

10
20
30
40
50
60
70
80
90
100
110
120
260
140

7

5 3 6 0 0 1 0 0 2 0 7 0 4 0 0

10
2 1 13
20
2 8 4
30
2 5 13
40
2 7 8
50
3 9 13 8
60
2 11 12
70
2 13 8

-3 1

-5 1 2

-1 13 100

-2 101

-3 102

-3 103

-1 8 200

-2 201

-3 250

-1 14 300

-4 5
2 11 12
2 13 9

-4 6
1 13
0

-5 50 250

-1 13 500
-1 13 501
-3 502
-2 503
-2 504
-3 505

-6
