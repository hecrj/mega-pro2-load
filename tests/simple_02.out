Servidores ocupados
0
Pelicula mas solicitada
0 0
Peticion procesada y servidores
0 3
5 7
Peticiones pendientes
0 13 100 103
Servidores ocupados
5 0 1
7 0 1
Servidores ocupados
0
Peticion procesada y servidores
1 1
2 5 7
Peticiones pendientes
0
Servidores ocupados
0
Peticion procesada y servidores
2 0
Pelicula mas solicitada
8 1
Peticion procesada y servidores
3 3
3 6
Peticion procesada y servidores
4 4
7
Servidores ocupados
3 3 1
6 3 1
7 4 3
Peticiones pendientes
4 13 501 505
Peticiones pendientes
4 13 501 505
Servidores ocupados
0
